schemas/JSON/settings/settings.schema.0.2.json

```diff
@@ -91,12 +91,24 @@
         }
       }
     },
+    "InstallRoot": {
+      "description": "The root directory where packages are installed to. Applies to the portable installer type.",
+      "type": "string",
+      "default":  "%LOCALAPPDATA%/WinGet"
```

Comment:
`%LOCALAPPDATA%\WPM_Packages` for user scope and `???` for machine scope.  If we are planning on supporting environment variables, we should call that out.